src/Command/ParseCalendarArchiveCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use App\Service\ArchiveParser;
+
+class ParseCalendarArchiveCommand extends Command
+{
+    private $parser;
+
+    public function __construct(ArchiveParser $parser, $name = null)
+    {
+        parent::__construct($name);
```

Comment:
@daria-sieroshtan 
Good girl!

Reply:
But I think you don't need "name" param

Reply:
This param is mandatory because of inheritance